docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`173` Add IOTA constant for cryptocompare
```

Comment:
Can you change it to something describing what is being fixed? So something like: "Price querying for IOTA should now properly work".